### PR TITLE
add merchant login

### PR DESCRIPTION
**Pull request by JanPietrzyk**

### CODE REVIEW

**Comment by shyim** [comment]
The API Controller is missing right? 🤔

**Comment by JanPietrzyk** [comment]
There is none, saleschannel-api/customer/login is sufficient

File: src/Merchants/Content/Merchant/SalesChannelContextExtension.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class SalesChannelContextExtension extends Struct
+{
+    private const NAME = 'merchant';
+
+    /**
+     * @var MerchantEntity
+     */
+    private $merchant;
+
+    public static function add(SalesChannelContext $salesChannelContext, MerchantEntity $merchant): void
```
Comment (PascalThesing) [inline src/Merchants/Content/Merchant/SalesChannelContextExtension.php]: I build this some minutes ago too...

Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/SalesChannelContextExtension.php]: sorry

**Pull request merged**